integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -41,8 +41,6 @@ COPY --from=crosscompile /opt/ubuntu-21.04-aarch64-sysroot /opt/ubuntu-21.04-aar
 
 COPY --from=ameba /opt/ameba /opt/ameba
 
-COPY --from=k32w /opt/sdk/sdks /opt/sdk/sdks
```

Comment:
This delete should be corrected together with the SDK path in ENV.

Reply:
It doesn’t build right now, at all. I’m trying to get things back to working.

Reply:
(K32W is too big to create a Docker image)